src/Appwrite/Platform/Tasks/PatchDeleteConsoleUsers.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace Appwrite\Platform\Tasks;
+
+use Appwrite\Event\Delete;
+use Utopia\Database\Document;
+use Utopia\Database\Helpers\ID;
+use Utopia\Platform\Action;
+use Utopia\CLI\Console;
+use Utopia\Database\Database;
+use Utopia\Database\Query;
+
+class PatchDeleteConsoleUsers extends Action
+{
+    public static function getName(): string
+    {
+        return 'patch-delete-console-users';
+    }
+
+    public function __construct()
+    {
+        $this
+            ->desc('Delete console users patch')
+            ->inject('dbForConsole')
+            ->inject('queueForDeletes')
```

Comment:
we should have a parameter for a --dry-run ( be default it needs to be true ) 

and have a confirmation where we have to type "YES" to proceed